css/mistica-common.css

```diff
@@ -1164,3 +1168,86 @@ button.mistica-display-card:has(.mistica-card__media):active:after {
         margin-top: 24px;
     }
 }
+
+/* Switch */
+.mistica-switch {
+    --switch-border-width: 2px;
+    --switch-transition: 300ms cubic-bezier(0.175, 0.885, 0.32, 1.275);
+    appearance: none;
+    position: relative;
+    width: 52px;
+    height: 32px;
+    border-radius: 16px;
+    background-color: var(--mistica-color-backgroundContainerAlternative);
```

Comment:
@aweell, we may need some changes in color constants. I tried the ones in [the spec](https://www.figma.com/design/J0LzwXn0tzGNIkgvFnt0Yz/%F0%9F%94%B8-Switch-Component-Specs?node-id=0-1004&t=wtuKa4zRYWAmik7I-0) and they don't work well, specially for the handle (bolita).

Also, these colors don't work well over inverse

Reply:
Discussed offline, spec pending to be updated to fix color issues

Reply:
Specs have been updated: https://www.figma.com/design/J0LzwXn0tzGNIkgvFnt0Yz/%F0%9F%94%B8-Switch-Component-Specs?m=auto&node-id=0-1004&t=QFFsCmhXQlZbtzQ9-1

We have decided removing the color change in the active handle hover state to avoid the need of creating new tokens.

I've been testing the spec changes in: https://mistica-web-jpho-mqzqgjrhh-alex-buenos-projects.vercel.app/mistica-css

Reply:
changes applied